drivers/mrf24j40/mrf24j40_internal.c

```diff
@@ -36,6 +36,56 @@ static inline void getbus(mrf24j40_t *dev)
     spi_acquire(SPIDEV, CSPIN, SPI_MODE_0, dev->params.spi_clk);
 }
 
+void mrf24j40_enable_pa_lna(mrf24j40_t *dev)
+{
+#if MRF24J40_USE_EXT_PA_LNA
+    /* Configure enable pin of the Voltage Regulator for the PA (GPIO3) on MRF24J40MC */
+    mrf24j40_reg_write_short(dev, MRF24J40_REG_TRISGPIO, 1 << 3);
+
+    /* Enable the volate regulator to power the Power Amplifier */
+    mrf24j40_reg_write_short(dev, MRF24J40_REG_GPIO, 1 << 3);
+
+    mrf24j40_reg_write_long(dev, MRF24J40_REG_TESTMODE, (MRF24J40_TESTMODE_RSSIWAIT0 |
+                                                         MRF24J40_TESTMODE_TESTMODE2 |
+                                                         MRF24J40_TESTMODE_TESTMODE1 |
+                                                         MRF24J40_TESTMODE_TESTMODE0));
+#else
+    (void) dev;
+#endif
+}
+
+void mrf24j40_disable_pa_lna(mrf24j40_t *dev)
+{
+#if MRF24J40_USE_EXT_PA_LNA
+    /* Disable automatic switch on PA/LNA */
+    mrf24j40_reg_write_long(dev, MRF24J40_REG_TESTMODE, MRF24J40_TESTMODE_RSSIWAIT0);
+
+    /* Configure all GPIOs as Output */
+    mrf24j40_reg_write_short(dev, MRF24J40_REG_TRISGPIO, 0xF);
+
+    /* Disable all GPIO outputs */
+    mrf24j40_reg_write_short(dev, MRF24J40_REG_GPIO, 0);
```

Comment:
Could it happen that other I/Os would be used for different purposes? If so, this line would disable them

Reply:
The other GPIOs are unused and not available through pads on the C/D/E modules, so unless you are building a custom module based on the bare mrf24j40 chip and add your own PA/LNA circuitry, this should be safe.

Reply:
Mhm, I see. But nevertheless, it seems cleaner if we only implement what the function name indicates: configure/disable pa and lna and nothing else. Mind to change?

Reply:
`enable_pa_lna()` will enable the voltage regulator for the PA and automatic PA/LNA control by the chip.
`enable_lna()` will disable the voltage regulator for the PA as well as automatic PA/LNA control, then manually enable the LNA.

Maybe I should call it `enable_auto_pa_lna()` to make that clearer.

Reply:
I like that name better. Question is here the same as below, why not read/modify **only** the bits you need/and write back?